dysnix/bsc/templates/servicemonitor.yaml

```diff
@@ -12,6 +12,16 @@ spec:
     port: {{ .Values.service.metricsPortName }}
     path: /metrics
     scheme: http
+  {{- if .Values.metricsExtra.enabled }}
+  {{- with .Values.metricsExtra }}
```

Comment:
I thought to do that but left as is so it will be matched to other servicemonitor configuration so we will have similar config options, cause chart large enough